_data/simple-icons.json

```
@@ -3522,6 +3522,12 @@
             "hex": "025E8C",
             "source": "https://dependabot.com"
         },
+        {
+            "title": "Depositphotos",
+            "hex": "000000",
+            "source": "https://blog.depositphotos.com/wp-content/uploads/2020/09/depositphotos_blog_logo.svg",
```

We try to avoid linking directly to files for the source as the run a higher risk of "link rot" when reviewing icons in the future.
```suggestion
            "source": "https://blog.depositphotos.com",
```